db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Role.find_or_create_by(name: 'observer')
+Role.find_or_create_by(name: 'client_admin')
+Role.find_or_create_by(name: 'admin')
+
+User.with_email_role_slug!(
+  ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
> When this was a migration, it didn't matter, because it only ran once

Based on all of the feedback in https://twitter.com/aidanfeldman/status/643550323538313216 and https://github.com/testdouble/feedback/issues/7, I'm leaning towards not thinking of migrations in that way. Also, almost everyone in those conversations advised not referencing models from migrations.

/cc https://github.com/18F/C2/pull/607 

---

What if we only create the records in `seeds` when the environment variables are set (or maybe use the defaults in non-production?), and print out a warning otherwise?